packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -1641,6 +1716,23 @@ export class Vector3 {
         return this;
     }
 
+	/**
+	 * Gets the rotation on all axes from the current vector to a vector
+	 * @param vector the target vector
+	 * @returns the rotation in the form (pitch, yaw, roll)
+	 */
+	public getRotationToVector(vector: DeepImmutable<Vector3>): Vector3 {
+		return Vector3.RotationBetweenVectors(this, vector);

```

Comment:
here RotationBetweenVectors is being used as a Vector3 method with two parameters

Line 2851 it is coded as a static with three parameters